cmake/Dependencies.common.cmake

```diff
@@ -199,3 +199,19 @@ add_library(cocoapi STATIC ${SOURCE_FILES})
 set_target_properties(cocoapi PROPERTIES POSITION_INDEPENDENT_CODE ON)
 list(APPEND DALI_LIBS cocoapi)
 list(APPEND DALI_EXCLUDES libcocoapi.a)
+
+##################################################################
+# RAPIDS Memory Manager
+##################################################################
+include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/third_party/rmm/include)
```

Comment:
Can you check if that compiler for QNX, aarch64 and SBSA?

Reply:
QNX passed.